src/Controls/samples/Controls.Sample/Pages/Others/GraphicsViewPage.xaml

```diff
@@ -28,6 +28,7 @@
                 EndInteraction="GraphicsView_EndInteraction"
                 StartInteraction="GraphicsView_StartInteraction"
                 StartHoverInteraction="GraphicsView_StartHoverInteraction"
+                MoveHoverInteraction="GraphicsView_MoveHoverInteraction"
```

Comment:
Naming the hover events with the "Move" word, should rename "DragInteraction" to "MoveInteraction"?